packages/web/hooks/use-swap.ts

```diff
@@ -531,3 +505,95 @@ function useSwapAsset(
     isLoading: isLoading && !existingAsset,
   };
 }
+
+/** Iterates over available and identical routers and sends input to each one individually.
+ *  Results are reduced to best result by out amount.
+ *  Also returns the number of routers that have fetched and errored. */
+function useQueryBestQuote(
+  input: RouterInputs["edge"]["quoteRouter"]["routeTokenOutGivenIn"],
+  enabled: boolean,
+  routerKeys = ["legacy", "sidecar", "tfm"] as RouterKey[]
+) {
+  const featureFlags = useFeatureFlags();
+  const availableRouterKeys: RouterKey[] = useMemo(
+    () =>
+      !featureFlags._isInitialized
+        ? []
+        : routerKeys.filter(
+            (key) => featureFlags.sidecarRouter || key !== "sidecar"
```

Comment:
I believe using a ternary can make it easier to read. 

```suggestion
            (key) => featureFlags.sidecarRouter ?  true : key !== "sidecar"
```